README.md

```diff
@@ -75,16 +96,18 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+app.get("/", function(req, res){
+  res.render("welcome");
+})
 
 ```
 
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
-```text
-Your answer...
+```
+Express and Rails are both backend frameworks however, they are quite different. Express uses the language, Javascript, and the database MongoDB while Rails uses the language, Ruby and the database, PostgresQL (for example). Express has an ODM (Object Data Mapper) called Mongoose and Rails has an ORM ( Object Relational Mapper) which is Active Record. Rails has a defined controller, model and route files, Express doesn't. Express uses handlebars for views while Rails used erbs.  
 ```
 
```

Comment:
Express actually doesn't have an ODM, in fact we have to bring in `mongoose` separately.

Otherwise, this is 👌